### PR TITLE
July Quarterly Update Blog

### PR DESCRIPTION
**Pull request by erogluorhan**
Whole blog has been written, and just a few corrections were made to the April quarterly update blog.

### CODE REVIEW

File: blog/_posts/2022-07-07-July-2022-update.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+We started to distribute the GeoCAT toolkits (GeoCAT-comp, GeoCAT-f2py, 
+GeoCAT-datafiles, and GeoCAT-viz) on conda-forge channel in this quarter. 
+Currently, we continue to distribute them via NCAR's Anaconda channel, too, but 
+our plan is to use only the conda-forge channel.
+
+Below you can find updates about our releases on several toolkits:
+
+#### - GeoCAT-comp and GeoCAT-f2py April Releases (v2022.04.0)!
+With these releases, there were some fixes such as making sure about the input 
+to output data type consistency, and some continuous integration improvements
```
Comment (hCraker) [inline blog/_posts/2022-07-07-July-2022-update.md]:
```suggestion
With these releases, there were some fixes such as ensuring input 
to output data type consistency, and some continuous integration improvements
```

This was a bit awkward to read before, but this small change should help

**Review by hCraker — CHANGES_REQUESTED** [review]
I made one suggestion for clarity and ease of reading. Other than that, this is good to go!

File: blog/_posts/2022-07-07-July-2022-update.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+#### UXarray Discussions
+
+Having input and feedback from our community is vital for the success of the 
+UXarray package (so Project Raijin). We encourage our community members to share 
+their feedback with us and give input to the UXarray project lifecycle. In 
+accordance with this, we created a Github Discussion, [Prioritization of Uxarray 
+analysis operators](https://github.com/UXARRAY/uxarray/discussions/46), to listen 
+to our community's priorities with the data analysis functions to be implemented. 
+The discussion has received a lot of feedback and led to conversation between the 
+folks. We are looking forward to implementing such functions as well as to keep
```
Comment (michaelavs) [inline blog/_posts/2022-07-07-July-2022-update.md]:
```suggestion
community members. We are looking forward to implementing such functions as well as to keep 

```
This edit could also encompass deleting "the" from the ending of line 36 so the sentence would read "...to conversation between [the] community members"

File: blog/_posts/2022-07-07-July-2022-update.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+- In addition, please refer to
+  [GeoCAT-comp API reference](https://geocat-comp.readthedocs.io/en/latest/api.html)
+  for detailed information about the GeoCAT-comp functions and how GeoCAT-f2py
+  is contained under GeoCAT-comp.
+
+#### - GeoCAT-viz April and May Releases (v2022.04.0 and v2022.05.0)!
+GeoCAT-viz has gone under a huge but backwards-compatible structural change 
+in [v2022.04.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.04.0) 
+and [v2022.05.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.05.0)
+that also has several new functionalities released! One of the biggest new
```
Comment (michaelavs) [inline blog/_posts/2022-07-07-July-2022-update.md]:
```suggestion
that also introduces several new functionalities! One of the major new 

```

File: blog/_posts/2022-07-07-July-2022-update.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+#### - GeoCAT-viz April and May Releases (v2022.04.0 and v2022.05.0)!
+GeoCAT-viz has gone under a huge but backwards-compatible structural change 
+in [v2022.04.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.04.0) 
+and [v2022.05.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.05.0)
+that also has several new functionalities released! One of the biggest new 
+functionalities is the capability to plot Taylor diagrams through a class 
+with the same name. The structural changes include moving the Contour plot 
+functionalities into a new class, Contour, which also enables tuning many 
+Contour plot-related arguments at the time of initiation. This helps the user 
+avoid setting those arguments by calling several functions separately, so it
```
Comment (michaelavs) [inline blog/_posts/2022-07-07-July-2022-update.md]:
```suggestion
avoid setting those arguments by calling several functions separately, thus 

```

File: blog/_posts/2022-07-07-July-2022-update.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+GeoCAT-viz has gone under a huge but backwards-compatible structural change 
+in [v2022.04.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.04.0) 
+and [v2022.05.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.05.0)
+that also has several new functionalities released! One of the biggest new 
+functionalities is the capability to plot Taylor diagrams through a class 
+with the same name. The structural changes include moving the Contour plot 
+functionalities into a new class, Contour, which also enables tuning many 
+Contour plot-related arguments at the time of initiation. This helps the user 
+avoid setting those arguments by calling several functions separately, so it 
+helps reduce the lines of codes significantly.
```
Comment (michaelavs) [inline blog/_posts/2022-07-07-July-2022-update.md]:
```suggestion
reducing the number of lines of code significantly.

```

**Review by michaelavs — APPROVED** [review]
Content looks good overall, just a couple of minor changes but I am approving currently so it can be merged once ready!

**Review by hCraker — APPROVED** [review]

**Pull request merged**